week-3/reading-list/script.js

```diff
@@ -21,3 +21,23 @@ const books = [
   },
 ];
 
+const ul = document.querySelector('#reading-list');
+for (const book of books) {
+  const li = document.createElement("li");
+  ul.appendChild(li);
+  //Display title of the book
+  const bookName = document.createElement("p");
+  bookName.innerText = book.title + "-------"+book.author;
+  li.appendChild(bookName);
+ //Display author of the book
+ //Display title of the book
+ const img = document.createElement("img");
+img.src = book.bookCoverImage;
```

Comment:
Can you make sure to run your code through a formatter like [prettier](https://prettier.io/) so it's consistent and easier to read?